packages/dataviews/src/dataviews.js

```diff
@@ -31,6 +31,7 @@ export default function DataViews( {
 	paginationInfo,
 	supportedLayouts,
 	onSelectionChange,
+	onDetailsChange = null,
```

Comment:
This feels to me just part of the `list` view.. Do you think it will have some kind of value in other views too? Maybe it's something that we should render in a field based on the view type? Maybe part of the layoutConfig?

Reply:
I did consider a different implementation: make it an action. From a conceptual point of view, it is an action. I looked into this, but I wasn't convinced it was worth it: it required connecting the "actions" and "view" APIs. The details only require a callback, everything else is provided by the list view already, so I thought of making that explicit.

I haven't considered implementing this as part of the fields or view APIs, but happy to hear your thoughts.